chia/rpc/data_layer_rpc_api.py

```diff
@@ -63,7 +63,8 @@ def get_routes(self) -> Dict[str, Callable[[Any], Any]]:
             "/delete_key": self.delete_key,
             "/insert": self.insert,
             "/subscribe": self.subscribe,
-            "/unsubscribe": self.unsubscribe,
+            "/get_kv_diff": self.get_kv_diff,
+            "/get_root_history": self.get_root_history,
```

Comment:
Should this be removing the `/unsubscribe` endpoint?  I see that `/get_root_history` isn't being added here but the missing endpoint entry is getting filled in.

```suggestion
            "/unsubscribe": self.unsubscribe,
            "/get_kv_diff": self.get_kv_diff,
            "/get_root_history": self.get_root_history,
```

Reply:
Thanks for catching, fixed